ktlint-cli-reporter-baseline/src/test/kotlin/com/pinterest/ktlint/cli/reporter/baseline/BaselineTest.kt

```diff
@@ -148,6 +155,10 @@ class BaselineTest {
 
             assertThat(logCaptor.errorLogs).contains("Unable to parse baseline file: $path")
         }
+
+        private fun setLocaleToEnglish() = Locale.setDefault(Locale.ENGLISH)
+
+        private fun resetLocale(default: Locale) = Locale.setDefault(default)
```

Comment:
When code is moved to BeforeEach and AfterEach, it is not necessary those helper methods. It is pretty obvious what is done.